stories/Templates/homepage/Homepage.js

```diff
@@ -18,21 +18,15 @@ import {
   LinkList
 } from '../../../src'
 
-import Header from '../shared/Header'
 import Footer from '../shared/Footer'
+import StickyHeader from '../../Header/StickyHeader'
```

Comment:
The component that you are replacing here had some logic for templates that your component doesn't have.
Rather than replace that, just add the `sticky` logic into the `shared/Header` component and leave this as it is.

Reply:
@dej611  Do you want me to remove the `StickyHeader.js` component I have created and instead make the `shared/Header` component sticky?

Reply:
Also, the `SlimHeader`, `NavHeader`, `CenterHeader` functions are class functions, should I convert them to functional type.

Reply:
Yes, make `shared/Header` sticky.
No, do not make it function component: there's a problem with Storybook and classes as the sourcecode is not shown.
There's already a function version of the complete header inspectable as a separate story, so when showing Templates we can assume the dev has another way to inspect it.